Updates to user research to add additional information (#145)
alphagov/forms: research/2022-06-28_Basic_Form_Building.md

> # Basic Form Building

Reviewer: ```suggestion
# Usability testing: Basic Form Building
```

> ## 2022-06-28 / Sprint 6

Reviewer: ```suggestion
2022-06-28 / Sprint 6
```

## Aims
To test usability of basic form builder journeys, including:
- Basic building a form journeys: 
- Name the form
- Add a long answer question and a date question (combination task)
- Add a short answer question 
- Add a National Insurance number question
- Add a phone number question
- Preview a question 

Return to a form journeys: 
- Change the name of a form 
- Change the content of questions 
- Change preview formatting
- Delete a question 
- Test a form 
- Re order questions

## Users
- Civil Servants
- 3 users

## Methodology
- Usability Testing

## Key Headlines

- There were consistently high scores across the tasks and some genuine moments of delight. Many elements of the prototype work well from a user perspective. However, there was also some familiar problems.
- The preview pane appeared to work more effectively than previous rounds. 

> - The addition of the 'preview and save' button was often used to discover the preview function. However, it suffers still from being visually recessive (especially adjacent to the vivid green 'save and continue button) and, for some, features too far down the page

Reviewer: ```suggestion
- The addition of the ‘preview and save’ button was often used to discover the preview function. However, it suffers still from being visually recessive (especially adjacent to the vivid green 'save and continue button) and, for some, features too far down the page
```
